### PR TITLE
Absolute image paths for the ProductInterface

### PR DESCRIPTION
**Pull request by eugene-shab**
### Description
This PR adds full product image url, path info and image label.

It should looks like:
```
small_image { url, label }
thumbnail { url, label }
image { url, label }
```
Regarding issue https://github.com/magento/graphql-ce/issues/174

### Contribution checklist
 - [x ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/graphql-ce?pullRequest=183) <br/>All committers have signed the CLA.

**Comment by naydav** [comment]
Pls, check tests
```

CategoryTest testCategoryProducts History | REST EE | < 1 sec
-- | -- | --
Magento\GraphQl\Catalog\CategoryTest::testCategoryProducts Exception: GraphQL response contains errors: Field "image" of type "ProductImage" must have a sub selection. Field "small_image" of type "ProductImage" must have a sub selection. Field "thumbnail" of type "ProductImage" must have a sub selection.   /opt/bamboo/builds/MCCE23-AT1255-RE/build-1/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/GraphQl/Client.php:105 /opt/bamboo/builds/MCCE23-AT1255-RE/build-1/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/GraphQl/Client.php:74 (2 more lines...)
  | Failed | CategoryTest testCategoryProducts History | SOAP EE | < 1 sec
Magento\GraphQl\Catalog\CategoryTest::testCategoryProducts Exception: GraphQL response contains errors: Field "image" of type "ProductImage" must have a sub selection. Field "small_image" of type "ProductImage" must have a sub selection. Field "thumbnail" of type "ProductImage" must have a sub selection.   /opt/bamboo/builds/MCCE23-AT1255-SOE/build-1/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/GraphQl/Client.php:105 /opt/bamboo/builds/MCCE23-AT1255-SOE/build-1/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/GraphQl/Client.php:74 (2 more lines...)
  | Failed | ProductViewTest testQueryAllFieldsSimpleProduct History | REST EE | < 1 sec
Magento\GraphQl\Catalog\ProductViewTest::testQueryAllFieldsSimpleProduct Exception: GraphQL response contains errors: Field "image" of type "ProductImage" must have a sub selection. Field "small_image" of type "ProductImage" must have a sub selection. Field "thumbnail" of type "ProductImage" must have a sub selection.   /opt/bamboo/builds/MCCE23-AT1255-RE/build-1/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/GraphQl/Client.php:105 /opt/bamboo/builds/MCCE23-AT1255-RE/build-1/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/GraphQl/Client.php:74
```

File: app/code/Magento/CatalogGraphQl/Model/Resolver/Product/ProductImage/Path.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+use Magento\Catalog\Model\Product;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+
+/**
+ * Returns product's image path
+ */
+class Path implements ResolverInterface
```
Comment (paliarush) [inline app/code/Magento/CatalogGraphQl/Model/Resolver/Product/ProductImage/Path.php]: No need to have this resolver, its logic can be moved to the main image resolver to gain better performance.

**Review by paliarush — CHANGES_REQUESTED** [review]

File: app/code/Magento/CatalogGraphQl/Model/Resolver/Product/ProductImage/Path.php (hunk 1 above)
Comment (naydav) [reply app/code/Magento/CatalogGraphQl/Model/Resolver/Product/ProductImage/Path.php]: Removed

**Review by naydav — APPROVED** [review]

**Pull request merged**